internal/testplanet/planet.go

```diff
@@ -116,34 +122,46 @@ func NewWithLogger(log *zap.Logger, satelliteCount, storageNodeCount, uplinkCoun
 
 	// init Satellites
 	for _, satellite := range planet.Satellites {
-		satellite.Kademlia.Service.SetBootstrapNodes(planet.nodeInfos)
+		satellite.Kademlia.Service.SetBootstrapNodes([]pb.Node{planet.Bootstrap.Local()})
 	}
 	// init storage nodes
 	for _, storageNode := range planet.StorageNodes {
-		storageNode.Kademlia.SetBootstrapNodes(planet.nodeInfos)
+		storageNode.Kademlia.SetBootstrapNodes([]pb.Node{planet.Bootstrap.Local()})
 	}
 
 	return planet, nil
 }
 
 // Start starts all the nodes.
 func (planet *Planet) Start(ctx context.Context) {
+	// First run the bootstrap nodes
 	for _, peer := range planet.peers {
-		go func(peer Peer) {
-			err := peer.Run(ctx)
-			if err == grpc.ErrServerStopped {
-				err = nil
-			}
-			if err != nil {
-				// TODO: better error handling
-				panic(err)
-			}
-		}(peer)
+		if peer.Local().Type == pb.NodeType_BOOTSTRAP {
```

Comment:
It should be fine to start everything in a single loop since they are started in a goroutine. This change doesn't really help that much.